src/sage/crypto/block_cipher/des.py

```diff
@@ -511,7 +512,7 @@ def encrypt(self, plaintext, key):
             sage: des.encrypt(P, K56) == C
             True
         """
-        if isinstance(plaintext, (list, tuple, Vector_mod2_dense)):
+        if isinstance(plaintext, (list, tuple, Vector)):
```

Comment:
are we sure that this change is safe for users ? I'm not sure it's currently tested...